### PR TITLE
manager-cloud-styles

### PR DESCRIPTION
**Pull request by frenautvh**
### 🚀 Feature
Legacy manager cloud styles

- commit history will be squashed on merge
- QC required on ovh-manager-cloud

### CODE REVIEW

File: packages/manager/modules/cloud-styles/package.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    "start": "lerna exec --stream --scope='@ovh-ux/manager-cloud-styles' --include-filtered-dependencies -- yarn run build",
+    "start:dev": "lerna exec --stream --scope='@ovh-ux/manager-cloud-styles' --include-filtered-dependencies -- yarn run dev",
+    "start:watch": "lerna exec --stream --parallel --scope='@ovh-ux/manager-cloud-styles' --include-filtered-dependencies -- yarn run dev:watch"
+  },
+  "devDependencies": {
+    "@ovh-ux/component-rollup-config": "5.0.0-beta.9",
+    "bootstrap": "~3.3.7",
+    "font-awesome": "^4.0.0",
+    "ovh-common-style": "^3.2.2",
+    "ovh-ui-kit": "^2.23.1"
```
Comment (antleblanc) [inline packages/manager/modules/cloud-styles/package.json]: Shouldn't be sync with the `package.json` from the Cloud Universe?

```json
    "ovh-ui-angular": "~2.24.0",
    "ovh-ui-kit": "~2.24.0",
```

File: packages/manager/modules/cloud-styles/src/deskaas/deskaas-details/deskaas-details.less [hunk 2]
```diff
@@ -0,0 +1,28 @@
+
```
Comment (antleblanc) [inline packages/manager/modules/cloud-styles/src/deskaas/deskaas-details/deskaas-details.less]: Extra empty line. Don't know if it was already  the case.

File: yarn.lock [hunk 3]
```diff
@@ -1512,7 +1512,7 @@
     rollup-pluginutils "^2.3.3"
     slash "^2.0.0"
 
-"@ovh-ux/component-rollup-config@^5.0.0-beta.9":
+"@ovh-ux/component-rollup-config@5.0.0-beta.9", "@ovh-ux/component-rollup-config@^5.0.0-beta.9":
```
Comment (antleblanc) [inline yarn.lock]: There is as duplicate.

File: packages/manager/modules/cloud-styles/package.json [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+    "build": "rollup -c --environment BUILD:production",
+    "dev": "rollup -c --environment BUILD:development",
+    "dev:watch": "yarn run dev --watch",
+    "prepare": "yarn run build",
+    "start": "lerna exec --stream --scope='@ovh-ux/manager-cloud-styles' --include-filtered-dependencies -- yarn run build",
+    "start:dev": "lerna exec --stream --scope='@ovh-ux/manager-cloud-styles' --include-filtered-dependencies -- yarn run dev",
+    "start:watch": "lerna exec --stream --parallel --scope='@ovh-ux/manager-cloud-styles' --include-filtered-dependencies -- yarn run dev:watch"
+  },
+  "devDependencies": {
+    "@ovh-ux/component-rollup-config": "5.0.0-beta.9",
```
Comment (antleblanc) [inline packages/manager/modules/cloud-styles/package.json]:
```suggestion
    "@ovh-ux/component-rollup-config": "^5.0.0-beta.9",
```

**Comment by cbourgois** [comment]
`LICENSE` file is missing (https://github.com/ovh-ux/manager/blob/master/LICENSE)

**Comment by frenautvh** [comment]
@antleblanc @cbourgois just updated the PR regarding your comments, thx for the review

**Pull request merged**